src/Servers/Kestrel/Core/src/Internal/KestrelServerImpl.cs

```diff
@@ -178,7 +178,7 @@ async Task OnBind(ListenOptions options, CancellationToken onBindCancellationTok
                         // Http/3 requires TLS. Note we only let it fall back to HTTP/1, not HTTP/2
                         else if (hasHttp3)
                         {
-                            throw new InvalidOperationException("HTTP/3 requires https.");
+                            throw new InvalidOperationException("HTTP/3 requires HTTPS.");
```

Comment:
? https is only defined as a url scheme and always uses lower case.

Reply:
But it is referred to in descriptions as upper case for its acryonm.

e.g. https://en.wikipedia.org/wiki/HTTPS